tests/genomics_tests/test_genomic_file_ingester.py

```diff
@@ -116,6 +116,37 @@ def test_replating_copy(self):
         self.assertEqual(copy_member.blockResearchReason, block_research_reason)
         self.assertEqual(copy_member.blockResearch, 1)
 
+    def test_validate_filenames(self):
+        job_controller = GenomicJobController(job_id=GenomicJob.AW1F_MANIFEST)
 
+        file_validator = GenomicFileValidator(
+            job_id=job_controller.job_id,
+            controller=job_controller
+        )
 
-
+        short_read_job_ids = [GenomicJob.AW1F_MANIFEST,
+                              GenomicJob.AW1_MANIFEST,
+                              GenomicJob.METRICS_INGESTION,
+                              GenomicJob.AW4_ARRAY_WORKFLOW,
+                              GenomicJob.AW4_WGS_WORKFLOW]
+
+        file_list = {
+            GenomicJob.AW1F_MANIFEST: ['UW_AoU_GEN_PKG-1234-567890_FAILURE.csv',
+                                       'UW_AoU_SEQ_PKG-1234-567890_FAILURE_v2.csv'],
+            GenomicJob.AW1_MANIFEST: ['UW_AoU_SEQ_PKG-1234-567890.csv',
+                                      'UW_AoU_GEN_PKG-1234-567890_v2.csv'],
+            GenomicJob.METRICS_INGESTION: ['UW_AoU_GEN_DataManifest_01234567_890.csv',
+                                           'UW_AoU_SEQ_DataManifest_01234567_890_v2.csv'],
+            GenomicJob.AW4_ARRAY_WORKFLOW: ['AoU_DRCB_GEN_2020-07-11-00-00-00.csv',
+                                            'AoU_DRCB_GEN_2020-07-11-00-00-00_v2.csv'],
+            GenomicJob.AW4_WGS_WORKFLOW: ['AoU_DRCB_SEQ_2020-07-11-00-00-00.csv',
+                                          'AoU_DRCB_SEQ_2020-07-11-00-00-00_v2.csv']
+        }
```

Comment:
So you already have your jobs id in the keys for this dictionary, no need to have them in the above list as well for the  main iteration in the test. 

Also I think it would be best to add valid and non-valid examples to the dictionary for iteration instead of only testing the correct filenames on most and incorrect ones on a select few. 

Maybe we can add those examples to the aforementioned dictionary and use the nested keys to determine if we should be asserting True of False.
Ex:
```
short_read_validation_map = {
   <job_id>: {
         'valid': [<valid examples>],
         'invalid': [<invalid examples>]
    }
}
```
This would encompass all your checks in a single iteration.